tests/sensors/test_ball_location_and_reward.py

```diff
@@ -32,6 +33,8 @@
     }
 
 
+@pytest.mark.skipif(holodeck.packagemanager.get_binary_path_for_package("Dexterity") is None,
```

Comment:
Great minds think alike ;) https://github.com/BYU-PCCL/holodeck/commit/6cba36610fc2c1078197ec42032b7abdef1fb96d

I think `"Dexterity" not in holodeck.installed_packages()` is a bit more readable, what do you think?

Reply:
I'll remove my commit

Reply:
Oh haha! I think we should keep your commit and get rid of this one since yours is pretty concise.

Reply:
@jaydenmilne I just looked at your commit and yeah, we should definitely go with that since it covers more cases and removes test_make. I'm closing this PR. Do you want to create a PR from `test-fixes`?